ad-joining/register-computer/join.ps1

```diff
@@ -27,13 +27,61 @@
 #--------------------------------------------------------------------------------------------
 
 $ErrorActionPreference = "Stop"
+$InformationPreference = "Continue";
+
+$metadataUri = "http://metadata.google.internal/computeMetadata/v1/instance";
+$metadataHeaders = @{"Metadata-Flavor" = "Google"};
+
+$enableDiagnostics = $false;
+try
+{
+    $enableDiagnostics = (Invoke-RestMethod -Headers $metadataHeaders `
```

Comment:
This only covers per-instance metadata, not per-project metadata -- I guess that's ok? Maybe add a comment.

Reply:
Correct. Do we think we should consider per-project metadata for this feature?

Reply:
Added per-project metadata option.